opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -590,7 +596,9 @@ mod tests {
             Ok(())
         }
 
-        fn shutdown(&mut self) {}
+        fn shutdown(&mut self) -> LogResult<()> {
```

Comment:
we can remove this implementation as the trait's default body is good enough here.

Reply:
🤦